<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolPage</class>
 <widget class="QWidget" name="ToolPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>0</y>
     <width>302</width>
     <height>206</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>View</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="4">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="decView">
         <property name="toolTip">
          <string>Decrease View</string>
         </property>
         <property name="toolTipDuration">
          <number>-1</number>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../icons.qrc">
           <normaloff>:/icons/zoom-out.png</normaloff>:/icons/zoom-out.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="fitView">
         <property name="toolTip">
          <string>Adjust to Content</string>
         </property>
         <property name="toolTipDuration">
          <number>-1</number>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../icons.qrc">
           <normaloff>:/icons/collapse.png</normaloff>:/icons/collapse.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QToolButton" name="incView">
         <property name="toolTip">
          <string>Increase View</string>
         </property>
         <property name="toolTipDuration">
          <number>-1</number>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../icons.qrc">
           <normaloff>:/icons/zoom-in.png</normaloff>:/icons/zoom-in.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="clearView">
         <property name="toolTip">
          <string>Clear View</string>
         </property>
         <property name="toolTipDuration">
          <number>-1</number>
         </property>
         <property name="text">
          <string>Clear View</string>
         </property>
         <property name="icon">
          <iconset resource="../icons.qrc">
           <normaloff>:/icons/sweep-512.png</normaloff>:/icons/sweep-512.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Connection</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QToolButton" name="refreshConnBtn">
         <property name="toolTip">
          <string>Refresh Connection</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../icons.qrc">
           <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
